app/code/Magento/MediaGalleryUi/Test/Mftf/Data/AdminEnhancedMediaGalleryImageData.xml

```diff
@@ -40,4 +40,13 @@
         <data key="extension">jpg</data>
         <data key="keywords">magento, mediagallerymetadata</data>
     </entity>
+    <entity name="LargeImage" type="image">
```

Comment:
I don't think new entity is needed. `ImageUpload` can be used (it's referencing the same file)